selfdrive/car/torque_data/override.yaml

```
@@ -57,6 +57,7 @@ KIA CARNIVAL 4TH GEN: [1.75, 1.75, 0.15]
 GMC ACADIA DENALI 2018: [1.6, 1.6, 0.2]
 LEXUS IS 2023: [2.0, 2.0, 0.1]
 KIA SORENTO HYBRID 4TH GEN: [2.5, 2.5, 0.1]
+HYUNDAI KONA 2ND GEN: [2.86, 2.5, 0.0969]
```

```suggestion
HYUNDAI KONA 2ND GEN: [2.5, 2.5, 0.1]
```

from `1618132d68afc876|2023-08-30--10-18-13`
`latAccelFactorRaw = 2.4725506, frictionCoefficientRaw = 0.101607`